cmd/skopeo/inspect.go

```diff
@@ -56,9 +57,13 @@ var inspectCmd = cli.Command{
 		if err != nil {
 			return err
 		}
+		var tag string
+		if tagged, ok := img.Reference().DockerReference().(reference.NamedTagged); ok {
+			tag = tagged.Tag()
+		}
 		outputData := inspectOutput{
 			Name: "", // Possibly overridden for a docker.Image.
-			Tag:  imgInspect.Tag,
+			Tag:  tag,
```

Comment:
```
+		var tag string
+		if tagged, ok := img.Reference().DockerReference().(reference.NamedTagged); ok {
+			tag = tagged.Tag()
+		}
```
So it can never be not OK?